caluma/caluma_core/health_checks.py

```diff
@@ -0,0 +1,213 @@
+from django.conf import settings
+from django.db import connections
+from django.db.migrations.executor import MigrationExecutor
+from django.db.migrations.autodetector import MigrationAutodetector
+from django.db.migrations.state import ProjectState
+from django.apps.registry import apps
+from django.http import JsonResponse
+from django.core import management
+
+from caluma.caluma_logging.models import AccessLog
+from caluma.caluma_form import storage_clients
+
+import watchman
+from watchman import views
+from watchman.decorators import check
+from watchman import settings as watchman_settings
+
+from io import BytesIO, StringIO
+import json
+
+
+@check
+def _check_pending_migrations(db):
+    """
+    Check database for pending migrations.
+    Returns JSON mapping if no pending migrations, otherwise raises Exception.
+    @check decorator catches and handles exception.
+    """
+
+    # check for changes i.e. migrations that haven't been made
+    # makemigrations --check exists with exit status 1 if changes exist
+    try:
+        management.call_command("makemigrations", "--check", "--dry-run")
```

Comment:
I think this bit could be left out, as it's enforced in CI anyway. If you run into this situation on a prod container, something's gone reeeeally wrong